Cubical/Categories/Displayed/Base.agda

```diff
@@ -88,6 +88,23 @@ module _ {C : Category ℓC ℓC'}
   ∫Cᴰ .⋆Assocᴰ _ _ _ = ΣPathP (_ , Dᴰ.⋆Assocᴰ _ _ _)
   ∫Cᴰ .isSetHomᴰ = isSetΣ Cᴰ.isSetHomᴰ (λ _ → Dᴰ.isSetHomᴰ)
 
+
+-- Display a category D over C
+-- Akin to weakening of a type theory
+module _ (C : Category ℓC ℓC') (D : Category ℓD ℓD') where
+  open Category
+  open Categoryᴰ
+  weaken : Categoryᴰ C ℓD ℓD'
```

Comment:
is this name too similar to `weakenᴰ`?

Reply:
If by weaken you mean "reindex" then unfortunately no as that leads to too many transports that don't reduce